LINQ/Program.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System.Collections;
+using System.Linq;
+
+internal class Program
+{
+    private static void Main(string[] args)
+    {
+        Console.WriteLine("..............Task 1...............");
+        List<int> ints = new List<int>() { 0, 4, 8, 12, 16 };
+        for (int i = 0; i < ints.Count; i++)
+        {
+            ints[i] /= 2;
+
+        }
+        var devidedByTwo = from s in ints
+                           where s % 2 == 0
+                           select s;
+        foreach (var s in devidedByTwo)
+        {
+            Console.Write(s + " ");
+
+        }
+        Console.WriteLine("\n..............Task 2...............");
+
+        List<int> ints2 = new List<int>() { 1, 3, -2, -4, -7, -3, -8, 12, 19, 6, 9, 10, 14 };
+        var positiveNumbers = from s in ints2
+                              where s > 0
+                              where s < 12
+                              select s;
+        foreach (var s in positiveNumbers)
+        {
+            Console.Write(s + " ");
+        }
+        Console.WriteLine("\n..............Task 3...............");
+
+        int[] array = new int[4] { 9, 8, 6, 5 };
+        var sqrNumbers = from s in array
+                         select "{ Number = " + s + ", SqrNo = " + (s * s) + " }";
+        foreach (var s in sqrNumbers)
+        {
+            Console.WriteLine(s);
+        }
+        Console.WriteLine("..............Task 4...............");
+
+        int[] array2 = new int[6] { 5, 5, 5, 9, 9, 1 };
+
+        var totalFives = array2.Count(s => s == 5);
+        var totalNines = array2.Count(s => s == 9);
+        var totalOnes = array2.Count(s => s == 1);
+
+        Console.WriteLine("The number and the Frequency are: ");
+        Console.WriteLine("Number 5 appeares {0} times ", totalFives);
+        Console.WriteLine("Number 9 appeares {0} times ", totalNines);
+        Console.WriteLine("Number 1 appeares {0} times ", totalOnes);
```

Comment:
I see copy-paste)) And now we know a lot of ways to avoid it. 
Soooooooo))))
1. Please create List of values which frequency in array you need to check (5, 9, 1)
2. Declare variable with type Dictionary<int, int> where key = number, value = frequency of number in array.
3. Initialise dictionary by using created list of numbers and LINQ method syntax.
4. Finally, you foreach for output the data from dictionary as `Number {key } appears {value} times`